go.work

```diff
@@ -0,0 +1,3 @@
+go 1.19
+
+use ./price-feeder
```

Comment:
let's remove this - we have old PR for this, and it's not related to this changset.

Reply:
Okay